dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.app.manager.wab.installer.fat;
+
+import static org.junit.Assert.fail;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import componenttest.custom.junit.runner.FATRunner;
+
+/**
+ *
+ */
+@RunWith(FATRunner.class)
+public class WabAdditionalTests extends AbstractWABTests {
+
+    private static final String SERVLET = "/WebDescriptorServlet";
+    private static final String OUTPUT_WEB_XML_SERVLET = "service: test.wab1.WebXmlMapped";
+
+    private static final String CONFIG = CONFIGS + "testDefault.xml";
+
+    @BeforeClass
+    public static void startServer() throws Exception {
+        server.setServerConfigurationFile(CONFIG);
```

Comment:
You should not need to set the server configuration file to the default `testDefault.xml` here.  The super class `AbstractWABTests.setUp()` method should have set this up with the default `testDefault.xml`  first before this method is called.

Reply:
Made suggested correction.